api/config_entry_routes.go

```diff
@@ -197,11 +197,12 @@ type HTTPQueryMatch struct {
 // HTTPFilters specifies a list of filters used to modify a request
 // before it is routed to an upstream.
 type HTTPFilters struct {
```

Comment:
(for a different file)
You'll also need to update the `api` file representing `ServiceRouter` to pick up the new change, as well as the one for the `v1/discovery-chain` endpoint response object.